src/pages/Scholarships.jsx

```diff
@@ -5,17 +5,19 @@ import {
   makeStyles,
   Typography,
 } from '@material-ui/core';
-import { createMuiTheme } from '@material-ui/core/styles';
 import Button from '@material-ui/core/Button';
 import Scholarships from '../models/Scholarships';
 import ScholarshipList from '../components/ScholarshipList';
 import FilterBar from '../components/FilterBar';
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme) => ({
   progress: {
     display: 'block',
     margin: 'auto',
   },
+  marginTB: {
```

Comment:
Ideally these class names would describe the element they style. Maybe `loadMoreButton` instead?